cmake/sdk_cmake_targets_readme.md

```diff
@@ -32,11 +32,8 @@ The remaining targets in `openenclave-targets.cmake` are automatically included
 when needed by the above.
 
 The package will also search for (and requires) the
-[`Threads` package](https://cmake.org/cmake/help/latest/module/FindThreads.html).
-
-However, the libraries `crypto` and `dl` will also need to be available on the
-system, but due to an outstanding issue, the failure will happen at link-time if
-they are unavailable.
+[`Threads` package](https://cmake.org/cmake/help/latest/module/FindThreads.html),
+`CryptoPP` package and `dlib` package.
```

Comment:
Can we link to both of these modules too?

Reply:
Links added. Please take a look at them and tell me if they are ok (I couldn't find any documentation for these two libraries on [cmake.org](https://cmake.org/) page).